upstream-community-operators/strimzi-kafka-operator/strimzi-cluster-operator.v0.13.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1052 @@
+              }
+            }
+        },
+        {
+            "apiVersion":"kafka.strimzi.io/v1beta1",
+            "kind":"KafkaUser",
+            "metadata":{
+              "name":"my-user",
+              "labels":{
+                  "strimzi.io/cluster":"my-cluster"
+              }
+            },
+            "spec":{
+              "authentication":{
+                  "type":"tls"
+              },
+              "authorization":{
+                  "type":"simple",
+                  "acls":[
+                    {
+                        "resource":{
+                          "type":"topic",
+                          "name":"my-topic",
+                          "patternType":"literal"
+                        },
+                        "operation":"Read",
+                        "host":"*"
+                    },
+                    {
+                        "resource":{
+                          "type":"topic",
+                          "name":"my-topic",
+                          "patternType":"literal"
+                        },
+                        "operation":"Describe",
+                        "host":"*"
+                    },
+                    {
+                        "resource":{
+                          "type":"group",
+                          "name":"my-group",
+                          "patternType":"literal"
+                        },
+                        "operation":"Read",
+                        "host":"*"
+                    },
+                    {
+                        "resource":{
+                          "type":"topic",
+                          "name":"my-topic",
+                          "patternType":"literal"
+                        },
+                        "operation":"Write",
+                        "host":"*"
+                    },
+                    {
+                        "resource":{
+                          "type":"topic",
+                          "name":"my-topic",
+                          "patternType":"literal"
+                        },
+                        "operation":"Create",
+                        "host":"*"
+                    },
+                    {
+                        "resource":{
+                          "type":"topic",
+                          "name":"my-topic",
+                          "patternType":"literal"
+                        },
+                        "operation":"Describe",
+                        "host":"*"
+                    }
+                  ]
+              }
+            }
+        }
+      ]
+    categories: "Streaming & Messaging"
+    certified: "false"
+    description: Strimzi provides a way to run an Apache Kafka cluster on Kubernetes or OpenShift in various deployment configurations.
+    containerImage: docker.io/strimzi/operator:0.13.0
+    repository: https://github.com/strimzi/strimzi-kafka-operator
+    createdAt: 2019-08-13T11:00:00Z
+    capabilities: Full Lifecycle
```

Comment:
We can set the capabilities to `Deep Insights` due to the availability of Prometheus metrics and Grafana dashboard for the operand referenced in the documentation. https://strimzi.io/docs/latest/#assembly-metrics-kafka-str

Reply:
@jmckind it's great to know that we can increase the capabilities but because the corresponding community operators PR was already merged, I would leave this change for the coming Strimzi 0.14.0 release in the future PRs. So if it is OK, you can just merge this one if there are no other problems.